clients/go/cmd/zbctl/main_test.go

```diff
@@ -89,6 +90,15 @@ var tests = []testCase{
 		goldenFile: "testdata/topology_json.golden",
 		jsonOutput: true,
 	},
+	{
+		name: "using json flag and host and port arguments",
+		cmd:  "status --output=json",
+		// we need to set the path so it evaluates $HOME before we overwrite it
+		envVars:        []string{fmt.Sprintf("%s=true", zbc.InsecureEnvVar), fmt.Sprintf("PATH=%s", os.Getenv("PATH"))},
+		goldenFile:     "testdata/topology_json.golden",
+		jsonOutput:     true,
+		useHostAndPort: true,
+	},
```

Comment:
Since this is just a configuration change and we don't really care about the interaction with the broker, we can replace this with a test in `client_test.go` which are faster